tests/compute/test_rpc.py

```diff
@@ -73,3 +147,22 @@ def test_rpc_msg():
     assert len(rpcmsg.data) == len(data)
     assert len(rpcmsg.tensors) == 1
     assert F.array_equal(rpcmsg.tensors[0], req.z)
+
+@unittest.skipIf(os.name == 'nt', reason='Do not support windows yet')
+def test_rpc():
+    ip_config = open("ip_config.txt", "w")
+    ip_config.write('127.0.0.1 30050 1\n')
```

Comment:
It will be great if we could test more combinations. You could create the task in the project tracker if you don't want to do so at the moment.